proto/responses.proto

```diff
@@ -173,3 +173,28 @@ message GetAccountStateDeltaResponse {
     // The calculated `AccountStateDelta` encoded using miden native format
     optional bytes delta = 1;
 }
+
+message GetAccountProofsResponse {
+    // Block number at which the state of the account was returned.
+    fixed32 block_num = 1;
+    // List of account state infos for the requested account keys.
+    repeated AccountProofsResponse account_state_infos = 2;
```

Comment:
Should the name of the field be `account_proofs` or something like that? (not sure if `state_infos` has much meaning in this context).